app/services/curation_concerns/working_directory.rb

```diff
@@ -1,10 +1,12 @@
 module CurationConcerns
-  module WorkingDirectory
+  class WorkingDirectory
```

Comment:
In case anyone wants to extend it. It was leftover from an earlier attempt with Sufia integration when I was trying to create:
`Sufia::WorkingDirectory < CurationConcerns::WorkingDirectory` I'm ambivalent about it now.